src/dragonball/src/api/v1/virtio_net.rs

```diff
@@ -29,8 +31,14 @@ pub enum Backend {
 }
 
 impl Default for Backend {
+    #[allow(unreachable_code)]
     fn default() -> Self {
-        Self::Virtio(VirtioConfig::default())
+        #[cfg(feature = "virtio-net")]
+        return Self::Virtio(VirtioConfig::default());
+        #[cfg(feature = "vhost-net")]
+        return Self::Vhost(VirtioConfig::default());
+
+        panic!("no available default network backend")
```

Comment:
Is panic a must here? We should carefully consider the use of `panic`

Reply:
Yes. But this statement should be unreachable as there is always a default value for networking, e.g. virtio-net.